src/software/SfM/main_evalQuality.cpp

```diff
@@ -28,98 +28,36 @@ int main(int argc, char **argv)
   CmdLine cmd;
 
   std::string
-    sGTDirectory,
-    sComputedDirectory,
-    sOutDir = "";
-  int camType = -1; //1: openMVG cam, 2,3: Strechas cam
+    sGTFile,
+    sComputedFile,
+    sOutDirectory = "";
 
 
-  cmd.add( make_option('i', sGTDirectory, "gt") );
-  cmd.add( make_option('c', sComputedDirectory, "computed") );
-  cmd.add( make_option('o', sOutDir, "outdir") );
-  cmd.add( make_option('t', camType, "camtype") );
+  cmd.add( make_option('i', sGTFile, "gt") );
+  cmd.add( make_option('c', sComputedFile, "computed") );
+  cmd.add( make_option('o', sOutDirectory, "outdir") );
 
   try {
     if (argc == 1) throw std::string("Invalid command line parameter.");
     cmd.process(argc, argv);
   } catch(const std::string& s) {
     std::cerr << "Usage: " << argv[0] << '\n'
       << "[-i|--gt] path (where ground truth camera trajectory are saved)\n"
```

Comment:
```
"[-i|--gt] ground truth path: it could be a json/abc file containing a reconstructed scene or localized camera, or the path to a directory containing the 2 other directories, "images" containing the images of the scene, and "gt_dense_cameras" containing the corresponding .camera files\n"
```
(check that what i say is correct)